From e1d2ec1a07217c3b3aced7449d5e4c1681cd9e0c Mon Sep 17 00:00:00 2001
From: Ahmad Samir <a.samirh78@gmail.com>
Date: Wed, 12 Jul 2023 21:10:35 +0300
Subject: [PATCH 18/81] Build with QT_NO_CONTEXTLESS_CONNECT

This disables the 3-arg QObject::connect() overload:
QObject::connect(sender, signal, functor)

For details see:
https://lists.qt-project.org/pipermail/development/2023-July/044141.html

No changes required.

Change-Id: If8875d9844a2fcf34c8a25e0f02d5927aff05c85
Reviewed-by: Volker Hilsheimer <volker.hilsheimer@qt.io>
---
 src/svg/CMakeLists.txt        | 1 +
 src/svgwidgets/CMakeLists.txt | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/svg/CMakeLists.txt b/src/svg/CMakeLists.txt
index 7b9c462..bb01c53 100644
--- a/src/svg/CMakeLists.txt
+++ b/src/svg/CMakeLists.txt
@@ -25,6 +25,7 @@ qt_internal_add_module(Svg
         qsvgtinydocument.cpp qsvgtinydocument_p.h
         qtsvgglobal.h qtsvgglobal_p.h
     DEFINES
+        QT_NO_CONTEXTLESS_CONNECT
         QT_NO_USING_NAMESPACE
     LIBRARIES
         Qt::CorePrivate
diff --git a/src/svgwidgets/CMakeLists.txt b/src/svgwidgets/CMakeLists.txt
index 8708799..004d486 100644
--- a/src/svgwidgets/CMakeLists.txt
+++ b/src/svgwidgets/CMakeLists.txt
@@ -10,6 +10,7 @@ qt_internal_add_module(SvgWidgets
         qsvgwidget.cpp qsvgwidget.h
         qtsvgwidgetsglobal.h
     DEFINES
+        QT_NO_CONTEXTLESS_CONNECT
         QT_NO_USING_NAMESPACE
     LIBRARIES
         Qt::CorePrivate
-- 
2.42.1

